itl/plugins-contrib.d/network-components.conf

```diff
@@ -833,3 +833,316 @@ object CheckCommand "printer_health" {
 	vars.printer_health_hostname = "$check_address$"
 	vars.printer_health_mode = "supplies-status"
 }
+
+template CheckCommand "generic-thola-check-command" {
+    import "plugin-check-command"
+
+    command = [ PluginContribDir + "/thola-client" ]
+
+    arguments += {
+        "(no key.0)" = {
+            order = -2
+            required = true
+            skip_key = true
+            value = "check"
+            description = "Thola Check Mode"
+        }
+        "--target-api" = {
+            required = true
+            value = "$thola_api_address$"
+            description = "Address of the thola API"
+        }
+    }
+}
+
+template CheckCommand "generic-thola-device-check-command" {
+    import "generic-thola-check-command"
+
+    arguments += {
+        "(no key.1)" = {
+            order = 0
+            required = true
+            skip_key = true
+            value = "$address$"
+            description = "Thola Check Device Mode"
+        }
+        "--snmp-community" = {
+            required = false
+            value = "$snmp_community$"
```

Comment:
Prefix all newly introduced custom vars for check commands with the check command name. E.g. `thola_device_snmp_community`.